platform/android/SCsub

```diff
@@ -55,7 +55,7 @@ if lib_arch_dir != "":
     else:  # release_debug, debug
         lib_type_dir = "debug"
 
-    out_dir = "#platform/android/java/lib/libs/" + lib_type_dir + "/" + lib_arch_dir
+    out_dir = "#platform/android/java/lib/libs/" + lib_type_dir + "/" + env["arch"]
```

Comment:
This should be `lib_arch_dir` instead of `env["arch"]` as the Android build system will look specifically for these ABIs terms.

Reply:
With the changes I made, this line expects `arm32` etc. Unless there are other places that are also incorrect. It's hard to tell specifically what is incorrect since it does compile fine (but just changing this one line to `lib_arch_dir` doesn't build if I recall correctly, I can retest later but I'm not available at the moment, maybe I just need to change what's in your other comment).

I do remember for sure that Line 64 does expect the values in `lib_arch_dir` as that points to sources in the NDK.

Reply:
It compiles fine, but I expect it to crash at runtime as the system will be unable to locate and load the relevant shared libraries.

Reply:
I changed this code and it builds fine again. Note that the export dialog doesn't work yet. That's a lot of work on its own so I'm considering either making that a second commit on this PR or making that be a follow-up PR.